README.md

```diff
@@ -15,7 +15,7 @@ From [Wikipedia](https://en.wikipedia.org/wiki/Emoji):
 
 ## Emoji support in JavaFX
 
-JavaFX can be used to create client applications that target desktop, mobiles and even web.
+JavaFX can be used to create client and embedded applications that target desktop, mobiles and even web.
```

Comment:
`create client applications that target desktop, embedded and mobile platforms and even web`?

Reply:
oops; thanks for the help